CHANGELOG/CHANGELOG-1.8.0.md

```diff
@@ -16,9 +16,9 @@
 
 ## Known Issues
 
-### CSI PowerStore: Not able to create ephemeral pods in OpenShift 4.13
+### CSI PowerFlex, CSI PowerStore, CSI Unity XT: Not able to create ephemeral pods in OpenShift 4.13
```

Comment:
Please update the table of contents with new heading

Reply:
Done

Reply:
Done